discovery-infra/test_infra/utils/utils.py

```diff
@@ -322,6 +322,37 @@ def wait_till_at_least_one_host_is_in_stage(
         raise
 
 
+def wait_till_specific_host_is_in_stage(
```

Comment:
This is kind of duplication of `wait_till_at_least_one_host_is_in_stage`. Maybe you can join them somehow instead of just duplicating the code?

Reply:
well, i can try to merge it to one method - the thing is that both methods doing different things
1 - wait for node_count hosts to get into specific stage, based on given nodes_count
2 - wait for specific 1 host to get into specific stage, based on given host 
isn't it be clearer to use on separated methods?

Reply:
Something like this

```suggestion
def wait_till_hosts_is_in_stage(
    client,
    cluster_id,
    stages,
    nodes_count,
    timeout,
    interval,
    hosts,
):
    log.info(f"Wait till {nodes_count} node is in stage {stages}")
    try:
        waiting.wait(
            lambda: are_host_progress_in_stage(
                hosts,
                stages,
                nodes_count,
            ),
            timeout_seconds=timeout,
            sleep_seconds=interval,
            waiting_for="Node to be in of the stage %s" % stages,
        )
    except BaseException:
        hosts = client.get_cluster_hosts(cluster_id)
        log.error(
            f"All nodes stages: "
            f"{[host['progress']['current_stage'] for host in hosts]} "
            f"when waited for {stages}"
        )
        raise


def wait_till_specific_host_is_in_stage(
    client,
    cluster_id,
    stages,
    host_name,
    nodes_count=1,
    timeout=consts.CLUSTER_INSTALLATION_TIMEOUT / 2,
    interval=5,
):
    hosts = [client.get_host_by_name(cluster_id, host_name)]
    return wait_till_hosts_is_in_stage(client, cluster_id, stages, nodes_count, timeout, interval, hosts)


def wait_till_at_least_one_host_is_in_stage(
    client,
    cluster_id,
    stages,
    nodes_count=1,
    timeout=consts.CLUSTER_INSTALLATION_TIMEOUT / 2,
    interval=5,
):
    hosts = client.get_cluster_hosts(cluster_id)
    return wait_till_hosts_is_in_stage(client, cluster_id, stages, nodes_count, timeout, interval, hosts)
```